.gitignore

```diff
@@ -4,6 +4,8 @@
 *.swp
 *.pyc
 
+*.travis.yml
```

Comment:
The travis file should not be there in the first place. I have not activated Travis CI.